toolkits/global/packages/global-forms/scss/00-tokens/_default.tokens.scss

```diff
@@ -0,0 +1,43 @@
+// Generated on 16/05/2022, 16:40:55
+// Source: design-tokens/componenets/global/global-forms/default.json
+// DO NOT edit directly
+
+$global-forms-checkbox-checked-background-color: $tokens--$background-container-inverted;
```

Comment:
There seems to be a rogue `$` in these names e.g. `$tokens--$background-container-inverted` should be `$tokens--background-container-inverted`.

Reply:
that'll be my bad JS skills at play. I'll take a look

Reply:
resolved in [this commit](https://github.com/springernature/frontend-toolkits/pull/746/commits/c1bb29c2f67085cbb7a9c6a5650dd80e954663e6)